sal_interfaces/HVAC/HVAC_Telemetry.xml

```diff
@@ -1,256 +1,1010 @@
+      <Description>estado_valvula</Description>
+      <IDL_Type/>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>caudal_ventilador_impulsion</EFDB_Name>
+      <Description>caudal_ventilador_impulsion</Description>
+      <IDL_Type/>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>comando_encendido</EFDB_Name>
+      <Description>comando_encendido</Description>
+      <IDL_Type/>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+  </SALTelemetry>
+  <SALTelemetry>
+    <Subsystem>HVAC</Subsystem>
+    <EFDB_Topic>HVAC_lsst_fwdsl_piso04_fwdsl_manejadora_fwdsl_sblanca</EFDB_Topic>
+    <item>
+      <EFDB_Name>estado_funcionamiento</EFDB_Name>
+      <Description>estado_funcionamiento</Description>
+      <IDL_Type/>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>lsst_fwdsl_piso04_fwdsl_manejadra_fwdsl_sblanca_fwdsl_estado_temperatura_ambiente</EFDB_Name>
+      <Description>lsst_fwdsl_piso04_fwdsl_manejadra_fwdsl_sblanca_fwdsl_estado_temperatura_ambiente</Description>
+      <IDL_Type/>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>estado_temperatura_exterior</EFDB_Name>
+      <Description>estado_temperatura_exterior</Description>
+      <IDL_Type/>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>estado_temperatura_anticongelante</EFDB_Name>
+      <Description>estado_temperatura_anticongelante</Description>
+      <IDL_Type/>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>valor_consigna</EFDB_Name>
+      <Description>valor_consigna</Description>
+      <IDL_Type/>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>setpoint_trabajo</EFDB_Name>
+      <Description>setpoint_trabajo</Description>
+      <IDL_Type/>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>temperatura_inyeccion</EFDB_Name>
+      <Description>temperatura_inyeccion</Description>
+      <IDL_Type/>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>temperatura_retorno</EFDB_Name>
+      <Description>temperatura_retorno</Description>
+      <IDL_Type/>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>alarma_general</EFDB_Name>
+      <Description>alarma_general</Description>
+      <IDL_Type/>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>alarma_filtro</EFDB_Name>
+      <Description>alarma_filtro</Description>
+      <IDL_Type/>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>reset_alarma</EFDB_Name>
+      <Description>reset_alarma</Description>
```

Comment:
Any idea what this is? It sounds more like a command than telemetry.

Reply:
To be honest, I don't know but according to the information that was given to me this is a genuine Telemetry topic.